src/renate/updaters/learner.py

```diff
@@ -76,6 +84,11 @@ def __init__(
         self._rng = get_generator(self._seed)
         self.save_hyperparameters(ignore=self._ignored_hyperparameters())
 
+    def _find_unique_classes_in_task(self):
```

Comment:
personally, I prefer if functions don't have side effects. Given that nothing is really happening in this function, maybe we can drop it and use `unique_classes` directly?

Reply:
Done.